content/en/blog/_posts/2019-01-14-apiserver-dry-run-and-kubectl-diff.md

```diff
@@ -21,7 +21,7 @@ experience with Kubernetes, and we tried to address some of these:
 
 - While compilers and linters do a good job to detect errors in pull-requests
   for code, a good validation is missing for Kubernetes configuration files.
-  The existing solution is to run `kubectl apply --dry-run`, but this runs a
+  The existing solution is to run `kubectl apply --dry-run=client`, but this runs a
```

Comment:
It's OK to leave the blog post unchanged; blog articles are written at a point in time. You could add some text at the _end_ of the blog post to tell readers that the syntaxt has changed since beta.

I'd prefer not to revise the main text of the article itself.

Reply:
Done! Added to the end of the blog post